src/graph.js

```diff
@@ -0,0 +1,52 @@
+import * as React from "react";
+import {Gitgraph} from "@gitgraph/react";
+//import { Gitgraph, Branch } from "@gitgraph/react";
+const gitgraph = require("@gitgraph/react")
+
+let master;
+
+class Graph extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            branches: [],
+            commitMessage:''
+        }
+    }
+
+
+    addCommit = () => {
+        // this.state.gitgraph.commit(this.state.commitMessage);
+        master.commit(this.state.commitMessage)
+    }
+
+    changeHandler = (event) => {
+        this.setState({
+            commitMessage: event.target.value
+        });
+    }
+
+    submitHandler = (event) => {
+        event.preventDefault();
+        this.addCommit();
+    }
+
+    render() {
+        return (
+            <div>
+                <form onSubmit={this.submitHandler}>
+                    <input type="text" value={this.state.commitMessage} onChange={this.changeHandler}/>
+                    <input type="submit" value="Commit on Life"/>
+                </form>
```

Comment:
I think the form can be it's own component. The code would be a bit more modular that way. Apart from that, looks good! Thanks for contributing! :D